packages/react/src/progress-bar/ProgressBar.recipe.ts

```diff
@@ -0,0 +1,33 @@
+import { createRecipe } from "../recipes";
+
+export const progressRootRecipe = createRecipe({
+  base: {
+    border: "1",
+    overflow: "hidden",
+  },
+  variants: {
+    size: {
+      default: { h: "6", w: "1/2" },
+    },
+    variant: {
+      default: {
+        bg: "white",
+      },
```

Comment:
@radnan bhaia, as I mentioned in the https://github.com/optimizely-axiom/optiaxiom/issues/49#issuecomment-2144331490, the given design for [Opal frontend](https://www.figma.com/design/GH25lYLDM34molSm1yhQO3/Opal-Guidelines?m=dev&node-id=2437-70026), we need another colour in (#7C3AED) to support that.
How can we support that?
Should we add another variant (maybe called `ai`?) to the recipe list? Or expose a prop for the `color`?

Reply:
I think, if the colour is base for the design, we should add it in our colour variant as semantic tokens, then we use it in our variants rather than exposing colour prop to always pass colour for the component.

Reply:
Thanks for the suggestion. However, as per the suggestion of @radnan bhaia, we are keeping the `bg.brand.solid` for now. And for the need of Opal frontend, we will modify the `--ax-colors-bg-success`variable providing a css `classname` as a prop to the component.

Reply:
As we are getting away from sprinkles, this sounds the right approach vai.